src/plugins/vis_builder/public/plugin.ts

```diff
@@ -70,13 +118,15 @@ export class VisBuilderPlugin
       id: PLUGIN_ID,
       title: PLUGIN_NAME,
       navLinkStatus: AppNavLinkStatus.hidden,
-      async mount(params: AppMountParameters) {
+      defaultPath: '#/',
+      mount: async (params: AppMountParameters) => {
         // Load application bundle
         const { renderApp } = await import('./application');
 
         // Get start services as specified in opensearch_dashboards.json
         const [coreStart, pluginsStart, selfStart] = await core.getStartServices();
         const { data, savedObjects, navigation, expressions } = pluginsStart;
+        this.currentHistory = params.history;
```

Comment:
Whats the difference between scoped and current history?

Reply:
They are the same thing. Setting the currentHistory to be params.history here since currentHistory is being used in `createOsdUrlTracker()`. 

```
createOsdUrlTracker({
      baseUrl: core.http.basePath.prepend(`/app/${PLUGIN_ID}`),
      ...
      getHistory: () => {
        return this.currentHistory!;
      },
```

we set scopedHistory to be currentHistory too when we set up the services:
```
const services: VisBuilderServices = {
          ...coreStart,
          scopedHistory: this.currentHistory,
          history: this.currentHistory,
          ...
```

Reply:
Hmm, if they are the same thing why do we need to use both?